src/atom.ts

```diff
@@ -0,0 +1,43 @@
+import { ConnectableObservable, Observable, Subject, Subscription } from 'rxjs';
+import { Lens } from '@types/ramda';
+import { Modifier } from './modifier';
+import { Projection } from './projection';
+import { LensedAtom } from './lensed-atom';
+
+export interface IAtom<T> {
```

Comment:
I would say, this declaration could be moved to separate file along with `Modifier` and `Projection` types, as those two may feel a bit lonely.